src/main/resources/assets/tconstruct/lang/en_us.json

```diff
@@ -3032,6 +3032,9 @@
 
   "jei.tconstruct.severing.title": "Severing",
 
+  "jei.tconstruct.tinkering.tool_building": "Tool Building",
+  "jei.tconstruct.tinkering.tool_building.anvil": "Broad tools require the Tinker's Anvil to be created or swap parts",
```

Comment:
Wonder if instead of "broad tools" we should just say "Tools with 4 or more parts" since the term "broad" is used kinda sparingly and that gives a more precise definition for players.